frontend/packages/ux-editor/src/components/config/editModal/EditOptions/EditCodeList/EditCodeList.tsx

```diff
@@ -31,6 +36,25 @@ export function EditCodeList<T extends SelectionComponentType>({
     });
   };
 
+  const { validateFileName, getDuplicatedOptionIds } = useValidateFileName(optionListIds);
+
+  const handleInvalidFileName = (file?: FormData, fileName?: string) => {
```

Comment:
`file` ser ikke ut til å være i bruk her.

Reply:
file variabelen var en del av mulig funksjonalitet for å overskrive en eksisterende fil dersom det er ønskelig.
Dette fjernet jeg siden funksjonaliteten ikke er en del av denne PRen, men er noe som kan tenkes å være nyttig senere.
Skal jeg hente inn denne funksjonaliteten igjen eller se om det er mulig å fjerne variabelen?

Reply:
Vi trenger i hvert fall ikke to parametere her - dette er enda et område hvor vi kan gjøre utbedringer på `StudioFileUploader`. Filnavnet er allerede tilgjengelig gjennom `file`; `file.get('file').name`. Jeg ville nok også kalt variabelen `formData` i stedet for `file`, for å gi den et riktigere navn.

Reply:
Jeg har fjernet variabelen fra EditCodeList.tsx, men har ikke endret det i StudioFileUploader.tsx fordi det vil resultere i duplisering av logikk for å hente fil navnet på alle steder som bruker StudioFileUploader.

Er endringene i commit [ae5ccbf](https://github.com/Altinn/altinn-studio/pull/13763/commits/ae5ccbfe3828a5f3eac56b28951aca5eb9afa708) bra. eller skal jeg gjøre endringene for alle filene som bruker komponenten?